prompts/2024/english/prompt_2024_01/AnaColon_summary_Bingham_and_Ratcliff_2024

```diff
@@ -0,0 +1,17 @@
+Complex multicellularity has evolved five separate times in eukaryotes but has never evolved in prokaryotes. Complex multicellularity has been defined as a large organism composed of many cells that lives for an extensive period of time. Some prokaryotes are multicellular but they do not have the morphologies of complex multicellular organisms. 
```

Comment:
I would say 'Complex multicellularity has been defined as a large organism composed of many cells that lives for an extensive period of time' is not *quite* accurate. The article's definition says that organisms with complex multicellularity *often* live long, but not that that's an essential part of the definition. They do say it's a 'term of art', so you've got some flexibility, but one important aspect missing from your definition is that these organisms don't just have many cells, but many cell *types*.